.github/workflows/ci-examples-app.yml

```diff
@@ -116,9 +116,10 @@ jobs:
         AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         AWS_DEFAULT_REGION: us-east-1
       run: |
+        echo $GITHUB_RUN_ID
         python -m coverage run --source ${{ env.COVERAGE_SCOPE }} \
-          -m pytest -m "not cloud" integrations_app \
-          --timeout=300 --durations=0 -vvvv
```

Comment:
Cut time to 90s as the longest test without hanging is only 40 so it does not make sense to wait so long...